python-stdlib/time/time.py

```diff
@@ -0,0 +1,37 @@
+import re
+
+_TS_YEAR=0
+_TS_MON=1
+_TS_MDAY=2
+_TS_HOUR=3
+_TS_MIN=4
+_TS_SEC=5
+_TS_WDAY=6
+_TS_YDAY=7
+_TS_ISDST=8
+
+_WDAY = ["Monday", "Tuesday", "Wednesday", "Thursday", "Friday",
+     "Saturday", "Sunday"]
+_MDAY = ["January", "February", "March", "April", "May", "June",
+     "July", "August", "September", "October", "November", "December"]
+
+_time_regex = re.compile(r"%(-?[a-zA-Z])")
```

Comment:
If be inclined to have this defined as None here, then calculated / set on first use in the function below to avoid the time / memory use during import.